### ISSUE TITLE
[rocm-bandwidth-test] installs broken symlink

### ISSUE THREAD

**Issue by zman0900** [rocm-arch/rocm-arch#67]
```
$ pacman -Ql rocm-bandwidth-test 
rocm-bandwidth-test /opt/
rocm-bandwidth-test /opt/rocm/
rocm-bandwidth-test /opt/rocm/rocm_bandwidth_test/
rocm-bandwidth-test /opt/rocm/rocm_bandwidth_test/bin/
rocm-bandwidth-test /opt/rocm/rocm_bandwidth_test/bin/rocm-bandwidth-test
rocm-bandwidth-test /usr/
rocm-bandwidth-test /usr/bin/
rocm-bandwidth-test /usr/bin/rocm_bandwidth_test

$ readlink -f /usr/bin/rocm_bandwidth_test
/opt/rocm/rocm_bandwidth_test/bin/rocm_bandwidth_test

$ readlink -ev /usr/bin/rocm_bandwidth_test
readlink: /usr/bin/rocm_bandwidth_test: No such file or directory
```

Notice the difference in dashes vs underscores. Program works fine if real path in /opt is used to run it, but link in /usr/bin is broken.

**Comment by acxz** [comment]
nice catch, sadly I don't have much time these days, but feel free to open up a PR!

**Comment by acxz** [comment]
@zman0900 sorry it took me so long to get to this but this issue should be fixed in `rocm-bandwidth-test-3.3.0-2`. Let us know if you still have any issues!
